examples/OrderEdit/StagedOrderTransitionStateAction.json

```diff
@@ -0,0 +1,8 @@
+{
+  "action": "transitionState",
+  "state":{
+    "typeId": "state",
+    "id": "{{stateId}}"
+  },
+  "force": true
```

Comment:
i would remove the `force` property. We should not promote force for transitioning states

Reply:
removed